frontend/src/views/Game.tsx

```diff
@@ -396,8 +390,8 @@ function GamePage() {
         setLoading(false);
 
         // Set the 'test' submission type to correctly display result.
-        submissions[submissions.length - 1].submissionType = SubmissionType.Test;
-        setCurrentSubmission(getSubmission(currentProblemIndex, submissions));
+        res.submissionType = SubmissionType.Test;
+        setCurrentSubmission(res);
```

Comment:
Yeah, those two lines specifically set the console to display the run code result. I had deleted the code before because I changed to the other system of storing runCodes.